test/.default_config

```diff
@@ -1,4 +1,3 @@
-tests.build-image-version=0.9
-tests.services-image-version=0.9
-tests.jobs-service-image-tag=quay.io/kiegroup/kogito-jobs-service:0.9.0-rc2
+tests.build-image-version=0.9.1-rc1
```

Comment:
this change is introduced for testing.
it will be removed once https://github.com/kiegroup/kogito-cloud-operator/pull/293 is merged